src/View/batchprocessing/ROINameCleaningOptions.py

```diff
@@ -318,16 +319,29 @@ def populate_table(self, dicom_structure, batch_directory):
             # Loop through each ROI in the RT Struct
             for i in range(len(rtstruct.StructureSetROISequence)):
                 # Get the ROI name
-                roi_name = rtstruct.StructureSetROISequence[i].ROIName
+                roi_name = rtstruct.StructureSetROISequence[i].ROIName.lstrip()
```

Comment:
.strip() rather then .lstrip() may be safer? let me know what u think

Reply:
Yea it would be, I used .lstrip() in my code because one of the Dicom files had whitespace at the start of the string. It totally past my mind that it could have whitespace at the end as well. I'll put this change in and resolve this shortly.